os/drivers/lcd/lcd_dev.c

```diff
@@ -319,6 +331,34 @@ static int lcddev_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 	return ret;
 }
 
+static int lcd_flushing_thread(void)
```

Comment:
I don't know this thread is necessary always. If not, we should introduce a new config and need to enable this for ST7701 only. @namanjain7 Please let me know your opinion on this.

Reply:
The flushing will never decrease performance. FPS may increase or remain same across lcd.
In case the performance remain same with and without flush, then there is no point in keeping this code. It will be good to just disable it for those lcd's.
I think it would be good to add a option for thread.